### PR TITLE
speed up a bit

### PR DESCRIPTION
**Pull request by jirfag**

### CODE REVIEW

**Comment by jirfag** [comment]
based on #1063

File: pkg/lint/lintersdb/enabled_set.go [hunk 1]
```diff
@@ -77,16 +77,22 @@ func (es EnabledSet) build(lcfg *config.Linters, enabledByDefaultLinters []*lint
...
-func (es EnabledSet) Get(optimize bool) ([]*linter.Config, error) {
+func (es EnabledSet) GetEnabledLintersMap() (map[string]*linter.Config, error) {
+	if err := es.v.validateEnabledDisabledLintersConfig(&es.cfg.Linters); err != nil {
+		return nil, err
+	}
+
+	return es.build(&es.cfg.Linters, es.m.GetAllEnabledByDefaultLinters()), nil
+}
+
+func (es EnabledSet) GetOptimizedLinters() ([]*linter.Config, error) {
```
Comment (ernado) [inline pkg/lint/lintersdb/enabled_set.go:88]: Please leave a comment on this method, it is not fully clear what `Optimized` means on first look.

Comment (jirfag) [reply pkg/lint/lintersdb/enabled_set.go:91]: thank you, I've left a comment

**Comment by jirfag** [comment]
tests timeouted on mac os only.

**Pull request merged**